### ISSUE TITLE
SA1515 reported on documentation headers

### ISSUE THREAD

**Issue by kiyote** [DotNetAnalyzers/StyleCopAnalyzers#2044]
If rule SA1514 is disabled, SA1515 is generated on documentation headers.

```
    #region Constructors
    /// <summary>
    /// Some summary text here.
    /// </summary>
    public SomeClass()
    {
```

SA1514 would have thrown due to the region being up against the documentation element, so we turned that off.  But then SA1515 started being thrown, and it's fairly clearly not a single line comment.

**Comment by sharwell** [comment]
Duplicate of #1371. :smile:

**Comment by sharwell** [comment]
Note: SA1515 is being reported due to a misconfigured build, where you set the compiler to treat all documentation comments as line comments. This was of course unintentional and it's even the default for a new project. Originally we created SA1652 to report this, but we are replacing it with a much more noticeable warning [SA0001](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA0001.md) starting with the 1.1 release.

**Comment by JoshuaLamusga** [comment]
@sharwell Is there a way to change this compiler behavior? I'm finding it difficult to discover, if so.

**Comment by sharwell** [comment]
@JoshuaLamusga See the documentation for SA0001 linked in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2044#issuecomment-171454858 for guidance on addressing this.

**Comment by JoshuaLamusga** [comment]
Yeah, I was just hoping it could be done without turning on XML doc generation.
